src/app/body/view-organization-details/members-access/members-access.component.ts

```diff
@@ -25,10 +32,74 @@ import { UserServiceService } from 'src/app/services/user-service/user-service.s
 export class MembersAccessComponent implements OnInit {
   @Input('members') members : MemberData[]
   memberRole: string
+  enableLoader:  boolean = false;
+  organizationDomain: string
+  membersEmail = []
+  team: Team;
+  user:User;
+  teamId: string;
+  memberCard: boolean = true;
+  deleteAccess: boolean = false;
+  componentName:string ="MEMBER-ACCESS";
 
-  constructor(public userService: UserServiceService, public rbaService: RBAService) { }
 
-  ngOnInit(): void {
+  constructor(public userService: UserServiceService, public rbaService: RBAService, private functions: AngularFireFunctions, private backendService: BackendService,   public errorHandlerService: ErrorHandlerService, private teamService: TeamServiceService, private route: ActivatedRoute) { }
+
+  ngOnInit(): void {    
+    if(this.rbaService.isAdmin == true) {
+      this.deleteAccess = true;
+    }
+    this.members.forEach((element)=>{ 
+      if(this.membersEmail.includes(element.Email)){
+      }else{
+        this.membersEmail.push(element.Email)
+      }
+      })
+  }
+
+  removeOrgMember(remove: string) {
+    this.enableLoader = true;
+    const callable = this.functions.httpsCallable('organization/removeOrgMember');
+    
+    if (this.organizationDomain == undefined) {
+      this.organizationDomain = this.backendService.getOrganizationDomain();
+    }
+    callable({OrganizationDomain: this.organizationDomain, OrgMembers: this.membersEmail, Remove: remove}).subscribe({
+      next: (data) => {
+        this.getOrgMembers(this.organizationDomain);
+        this.enableLoader = false;
+        const index = this.membersEmail.indexOf(remove);
+        if (index != -1) {
+          this.membersEmail.splice(index, 1);
+          this.team.TeamMembers.splice(index, 1);
```

Comment:
this.team will be undefined and there is no need for this variable can be removed